.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,68 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+    # will start the job at 06:00 UTC every Monday
+    - cron: "0 6 * * 1"
```

Comment:
this one should be Okay I think since Lahore time would be UTC + 5 and (just an FYI) we want the upgrade PRs to be ready before workday for Lahore starts